test/unit/ReservoirPriceOracle.t.sol

```diff
@@ -621,6 +622,31 @@ contract ReservoirPriceOracleTest is BaseTest {
         assertEq(lPriceAB, 0); // composite price is not stored in the cache
     }
 
+    function testUpdatePrice_RecipientOutOfGas() external {
+        // arrange
+        GasBuster lGasBuster = new GasBuster();
+
+        _writePriceCache(address(_tokenA), address(_tokenB), 5e18);
+        deal(address(_oracle), 1 ether);
+
+        skip(1);
+        _pair.sync();
+        skip(_oracle.twapPeriod() * 2);
+        _tokenA.mint(address(_pair), 2e18);
+        _pair.swap(2e18, true, address(this), "");
+
+        // act
+        _oracle.updatePrice(address(_tokenB), address(_tokenA), address(lGasBuster));
+
+        // assert
+        (uint256 lPrice,) = _oracle.priceCache(address(_tokenA), address(_tokenB));
+        assertEq(lPrice, 98_918_868_099_219_913_512);
+        (lPrice,) = _oracle.priceCache(address(_tokenB), address(_tokenA));
```

Comment:
This won't work on mainnet because the TX gas limit will be much lower then I think it is in the test

Reply:
okay the test used about 10 billion gas while the block gas limit is 30M on mainnet right now. 

So do you have any tips on what to do? Should I modify the test case to simulate the mainnet gas limit, or something else?

Reply:
Well, this test will fail on mainnet right. So I'm not sure what value there is in it because we are not trying to protect against an mev bot that wastes all the gas, that's their problem. We're trying to just keep it as simple as possible.

So I think so long as you have tests already showing that a bot can get paid if they play nice, we don't need this gas bomb contract + test.

Reply:
okay rm-ed test case and associated gas-waster